doc/start/intro.rst

```diff
@@ -2,14 +2,19 @@
  Intro to Ceph
 ===============
 
-Whether you want to provide :term:`Ceph Object Storage` and/or
-:term:`Ceph Block Device` services to :term:`Cloud Platforms`, deploy
-a :term:`Ceph File System` or use Ceph for another purpose, all
-:term:`Ceph Storage Cluster` deployments begin with setting up each
-:term:`Ceph Node`, your network, and the Ceph Storage Cluster. A Ceph
-Storage Cluster requires at least one Ceph Monitor, Ceph Manager, and
-Ceph OSD (Object Storage Daemon). The Ceph Metadata Server is also
-required when running Ceph File System clients.
+Ceph can be used to provide :term:`Ceph Object Storage` to :term:`Cloud
+Platforms` and Ceph can be used to provide :term:`Ceph Block Device` services
+to :term:`Cloud Platforms`. Ceph can be used to deploy a :term:`Ceph File
+System`.  All :term:`Ceph Storage Cluster` deployments begin with setting up
+each :term:`Ceph Node` and then setting up the network. 
+
+A Ceph Storage Cluster requires the following: at least one Ceph Monitor (and
+one Ceph Manager per monitor), and as many Ceph OSDs as there are copies of an
```

Comment:
Mgrs are not tied to mons this way, though it’s best practice.  Rook does not allow deploying more than two total.

Reply:
Did you take out the mgr altogether?  To be clear I was suggesting at least one mon and at least one mgr.  Their placement and numbers are not coupled in any way.

Reply:
Crud. Serves me right for working while jetlagged.